packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,487 @@ Before installing Chainweb, you should verify that your computer meets the follo
+   Before you start the node, you should review the configuration options and the default values to determine whether you want to make any changes to the configuration of the node.
+   For information about this next step, see [Review the default configuration](#review-the-default-configuration).
+
+### Build with Nix
+
+One advantage of using the Nix package manager to build and run the chainweb-node binary is that Nix caches binary dependencies, so you can download pre-built binaries for for the libraries and packages that Chainweb requires.
+
+To build with the Nix package manager:
+
+1. Download and install [Nix](https://nixos.org/nix/) by clicking **Get Nix**, then follow the instructions.
+
+1. Open your shell startup profile file for the shell you use in a text editor.
+   
+   For example, if you are using the `bash` shell, open the `.bash_profile` file.
+   If you are using the `zsh` shell, open the `.zsh_profile` file.
+
+1. Add the following line to the startup profile:
+
+   ```text
+   . $HOME/.nix-profile/etc/profile.d/nix.sh
+   ```
+
+   If you don't want to edit the profile directly using a text editor or don't have a text editor installed, you can run a command similar to the following:
+   
+   ```bash
+   echo ". $HOME/.nix-profile/etc/profile.d/nix.sh" >> ~/.bash_profile
+   ```
+   
+   If you are using a different shell, change `.bash_profile` to the appropriate startup script for your shell.
+
+1. Open the `/etc/nix/nix.conf` file in a text editor and add the following lines to the file:
+
+   ```text
+   substituters = https://nixcache.chainweb.com https://cache.nixos.org/
+   trusted-public-keys = nixcache.chainweb.com:FVN503ABX9F8x8K0ptnc99XEz5SaA4Sks6kNcZn2pBY= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+   experimental-features = nix-command flakes
+   ```
+
+   If the `/etc/nix` directory doesn't exist, you should switch to the root user and create it, then add the `nix.conf` file to the directory.
+
+2. Restart the nix-daemon.
+   
+   Most installations of Nix are multi-user and require you to restart the nix daemon to make your `nix.conf` changes take effect. 
+   
+   You can check whether Nix is running in multi-user or single-user mode by running the following command:
+   
+   ```bash
+   ps aux | grep nix-daemon
+   ```
+   
+   If you see only the `grep` command, then you are using single-user mode and you don't have to do anything else. 
+   If you see a `nix-daemon` process, then you are using multi-user mode and you need to restart the process
+   
+   On macOS, run the following commands:
+   
+   ```bash
+   sudo launchctl stop org.nixos.nix-daemon
+   sudo launchctl start org.nixos.nix-daemon
+   ```
+   
+   On Linux, run the following command:
+   
+   ```bash
+   sudo systemctl restart nix-daemon.service
+   ```
+
+1. Change to the directory that contains the source code for the `chainweb-node` binary.
```

Comment:
This is unnecessary and not recommended. Nix does not require you to clone the source repository.

Users should just run

```
nix build github:kadena-io/chainweb-node/<tag>
```

where tag is the released version, eg 2.24.1